Fixed image sizes + updated description (#27)
xataio/mdx-blog: replica-identity-full-performance.mdx

---

> title: "On the performance of REPLICA IDENTITY FULL in Postgres"

Reviewer: ```suggestion
title: "Improve the performance of REPLICA IDENTITY FULL in Postgres"
```

Author: We're not really improving it, just discussing it.

> description: "Analyzing the performance impact of REPLICA IDENTITY FULL by looking at the source code and performing benchmarks."

Reviewer: ```suggestion
description: "How to analyze replica performance by looking at your source code and performing benchmarks."
```

Author: I meant the Postgres source code, actually. Let me clarify that.

Author: REPLICA IDENTITY FULL is the setting name in Postgres. It doesn't have to be capitalized, but I think most people will recognize it like that, because that's how you would typically set it:

```
ALTER TABLE foo REPLICA IDENTITY FULL;
```
